python/pyspark/ml/torch/distributor.py

```diff
@@ -514,11 +514,26 @@ def _execute_command(
                 f"Command {cmd} failed with return code {task.returncode}. "
                 f"The {last_n_msg} included below: {task_output}"
             )
+    @staticmethod  
+    def _get_output_from_framework_wrapper(framework_wrapper: Optional[Callable], input_params: Dict, train_object: Union[Callable, str], run_pytorch_file_fn: Optional[Callable], *args, **kwargs) -> Optional[Any]:
+        if not framework_wrapper:
+            raise RuntimeError("In the _get_output_from_framework_wrapper function, found a framework wrapper that is none. I wonder why this is...")
+        # The object to train is a file path, so framework_wrapper is some run_training_on_pytorch_file function.
+        if type(train_object) is str:
+            return framework_wrapper(input_params, train_object, *args, **kwargs)
+        else:
+            # We are doing training with a function, will call run_training_on_pytorch_function
+            if not run_pytorch_file_fn:
+                run_pytorch_file_fn = TorchDistributor._run_training_on_pytorch_file
+            return framework_wrapper(input_params, train_object, run_pytorch_file_fn, *args, **kwargs)
+
+        
 
     def _run_local_training(
         self,
         framework_wrapper_fn: Callable,
         train_object: Union[Callable, str],
+        run_pytorch_file_fn: Optional[Callable],
```

Comment:
Id probably move this variable before train_object

Reply:
do you mind if I ask why?

Reply:
This is just a `nit` but we want to keep the code that relates to the training (train_object, *args, **kwargs) away from the utils stuff like `framework_wrapper_fn` and `run_pytorch_file_fn` for the sake of readability.

Reply:
when you say other functions, do you mean all of them? Wouldn't that interfere with the default args comment because they have to be after positional args iirc?

Reply:
Yes, `run_pytorch_file_fn` isn't a keyword argument (yet); you can either do this comment or the other, but I'd prefer Weichen to weigh in first.

Reply:
ok sounds good!